maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm

```diff
@@ -8,7 +8,8 @@
 	spawn_weight = 1
 	suffixes = list("away_site/abandoned_industrial/abandoned_industrial_station.dmm")
 
-	sectors = list(ALL_POSSIBLE_SECTORS)
+	sectors = list(ALL_TAU_CETI_SECTORS, ALL_COALITION_SECTORS, SECTOR_BADLANDS, SECTOR_VALLEY_HALE, SECTOR_GAKAL, SECTOR_LIGHTS_EDGE)
```

Comment:
why?

Reply:
using ALL_POSSIBLE_SECTORS is bad, as it means future people will have to go through and blacklist sectors that it doesn't fit individually, as well as existing sectors that won't have a lot of human ruins such as the lemurian sea

Reply:
It is not bad, and that is fully intentional, and better than the alternative. Reasoning being:
- It is very easy to notice that a site/ship spawns in a sector, when it should not be spawning. You just look at the possible sites in current sector, that I added to show around round start.
- It is very hard to notice that a site/ship should be spawning in a sector, but it does not. You have to go through all the files and folders of ships and sites, and we have a ton of those, and only getting more and more.

The whole point of definitions like `ALL_POSSIBLE_SECTORS` or `ALL_COALITION_SECTORS` is to not have to go through every new ship/site to add them to whatever new sector that is being added. Like how it used to be in the past before I made `sectors` to allow nested lists, and added `sectors_blacklist`.

Reply:
So yeah. Please keep it `ALL_POSSIBLE_SECTORS` and blacklist from any sectors it should not spawn in.

Also. The blacklist can have nested lists too, so in the future there could be defines like `ALL_SECTORS_WITH_NO_HUMAN_DERELICTS` for example, so any human derelicts would just have a `sectors_blacklist = list(ALL_SECTORS_WITH_NO_HUMAN_DERELICTS)`. Or the same situation with defines like `ALL_CIVILIZED_SECTORS` or something that currently would hold Tau Ceti and Haumenin.